### DISCUSSION TITLE
new to arch linux. what are your recommended IgnorePaths?

### DISCUSSION THREAD

**Discussion by tryprncp** [CyberShadow/aconfmgr#212] [Q&A]
Hey guys, new to arch linux and linux in general. what are your recommended IgnorePaths?
I have tried running `aconfmgr save` and it recommend a lot of directories that I may include in IgnorePaths. What are your suggestion about this?

**Comment by CyberShadow** [comment]
You could have a look at some existing acongmgr configurations on GitHub:

https://github.com/search?q=IgnorePath+language%3Abash+path%3Aaconfmgr&type=code

**Comment by izmyname** [comment]
Also, here's my configuration with the focus on inlining files, rather than copying them, for reference. https://github.com/izmyname/aconfmgr

**Reply by hcartiaux** [reply]
I've spent some time on my configuration too and [here is my ignore list](https://github.com/hcartiaux/aconfmgr/blob/main/10-ignore.sh)
